Issue during build on the latest Go client for Kubernetes

On attemp to build _knctl_ I have issues with the go-client for _kubernetes_ i.e 
 ```
 $ dep ensure
 $ hack/build.sh                                                                                                         git:(master|✚5… 
	+ go fmt ./cmd/... ./pkg/... ./test/...
	+ go build ./cmd/...
	# github.com/ipolyzos/knctl/vendor/k8s.io/client-go/plugin/pkg/client/auth/azure
	vendor/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go:246:4: cannot use expiresIn (type string) as type json.Number in field value
	vendor/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go:247:4: cannot use expiresOn (type string) as type json.Number in field value
	vendor/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go:248:4: cannot use expiresOn (type string) as type json.Number in field value
	vendor/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go:265:23: cannot use token.token.ExpiresIn (type json.Number) as type string in assignment
	vendor/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go:266:23: cannot use token.token.ExpiresOn (type json.Number) as type string in assignment
```

The issue seems to be caused on the Azure Active Directory plugin for client authentication part.

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -31,3 +31,11 @@
   name = "k8s.io/code-generator"
   unused-packages = false
   non-go = false
+
+# Fix: #49
+#
+# NOTE:
+#   use of go-autorest v10.14.0
+[[override]]
+name = "github.com/Azure/go-autorest"
+revision = "bca49d5b51a50dc5bb17bbf6204c711c6dbded06"

